approaches/bulk-data-loading/src/GqlMutations.js

```diff
@@ -0,0 +1,85 @@
+export class GqlMutations {
+
+    static createInventoryListQuery() {
+        let query = `
+        query ListInventoryItems {
+          list_InventoryItems(readMode: NODE_LEDGERED) {
+            _InventoryItems {
+              _id
+            }
+            nextToken
+          }
+        }
+    `;
+
+        return {
+            query: query,
+            variables: {}
+        }
+    }
+
+    static createInventoryListNextPageQuery(nextToken) {
+        let query = `
+        query ListInventoryItems($nextToken: String) {
+          list_InventoryItems(nextToken: $nextToken, readMode: NODE_LEDGERED) {
+            _InventoryItems {
+              _id
+            }
+            nextToken
+          }
+        }
+    `;
+
+        return {
+            query: query,
+            variables: {
+                nextToken: nextToken
+            }
+        }
+    }
+
+    static createInventoryMutation(batch) {
```

Comment:
We should be able to support this in the client SDK (sorry that we don't yet). If we can land on a good abstraction, then I don't think it would be too hard to implement.

Reply:
A method that looks like this `entities.inventory.add([array,of,inventory,items])` would be ideal (producing an array of result objects).  The string concatenation I'm doing in GqlMutations.js would then be left to the SDK, not the SDK user.